src/translib/transformer/xfmr_vrf.go

```diff
@@ -202,13 +240,10 @@ var network_instance_table_name_xfmr TableXfmrFunc = func (inParams XfmrParams)
                 return tblList, errors.New("Network instance not set")
         }
 
-        pathInfo := NewPathInfo(inParams.uri)
+        pathInfo := NewPathInfoFirstKey(inParams.uri)
```

Comment:
Not sure why we need this change? dont we have lib function already?

Reply:
@venkatmahalingam - Details are shared in email for this. Is this question for the API or "firstKey"?

Reply:
Hi Venkat,

As Bhavini mentionted, it's the detail explained in the email (again below), 
2.	While debugging the other brcm issue (GET igmp), Kwan found the network instance top level table name transformer is returning the incorrect table name( igmp table name)
This is because the inParams.uri received by the table name transformer function points to the requestUri path (which has multiple name keys), rather than the uri pointing to the top level network instance.
The existing NewPathInfo(inParams.uri) seems to be returning the last name key only. I copied the function inside xfmr_vrf.go,  renamed and changed it to return the first name key. Tried few other cases where table name transformer function is not triggered but only the key transformer function, and there the inParams.uri is correct.  So I only changed it for table name transformer.
Ideally the inParams.uri passing to table name transformer should be correct. 

Thanks,
Bing